integration-test/tests/save-prompts.android.spec.js

```diff
@@ -124,4 +124,60 @@ test.describe('Android Save prompts', () => {
             await login.promptWasNotShown()
         })
     })
+
+    test.describe('Prompting to save from a poor login form (using Enter and click on a button outside the form)', () => {
```

Comment:
There's a lot of repetition in these tests where the steps aren't much different and only the setup and assertion differ slightly. We should consider writing a generic test that takes the setup as a parameter. Something for another day.